src/RenderManager.cpp

```diff
@@ -0,0 +1,25 @@
+#include "RenderManager.h"
+#include "RenderObject.h"
+#include <algorithm>
+
+
+void RenderManager::renderAll()
+{
+	if (!this->isSorted) {
+		std::ranges::sort(this->renderObjects, RenderObject::ZIndexComparator());
+	}
+	std::ranges::for_each(this->renderObjects, [](RenderObject* obj) {
```

Comment:
unnecessary use of for_each resulting in the import of `<algorithm>` maybe change to standard for loop

Reply:
Has been changed for readability, however the `<algorithm>`-library is still required for `std::sort` and `std::remove`.